### ISSUE TITLE
Wrong toolbar tabbing order

### ISSUE THREAD

**Issue by Noordfrees** [widelands/widelands#4233]
The tabbing order in the InteractivePlayer's, the InteractiveSpectator's, and the Editor toolbar is messed up because the respective buttons/dropdowns are being created in the wrong order.

The tabbing order for a simple line of menu icons should always be left-to-right.

**Comment by frankystone** [comment]
> The tabbing order for a simple line of menu icons should always be left-to-right.

Maybe the order should be dependent with regard to the language: People used to read from right to left may need a different ordering.

**Comment by gunchleoc** [comment]
The tabbing order should still be the same in that case, but the icons display order reversed.

Swapping the display for RTL should be done in the `UI::Box` class once we have converted everything to box layout.
